minimum-constraints.txt

```diff
@@ -53,7 +53,8 @@ click-spinner==0.1.8
 click-repl==0.1.6
 asciitree==0.3.3
 tabulate==0.8.2
-prompt-toolkit==2.0.1
+prompt-toolkit==1.0.15; python_version == '2.7'
+prompt-toolkit==2.0.1; python_version >= '3.4'
```

Comment:
This same distinction  also needs to be made in minimum-constraints.txt

Reply:
Getting late.. that is minimum-constraints.txt

Reply:
Correct on both points. It was indeed late ...